src/VirtoCommerce.NotificationsModule.Data/Migrations/NotificationDbContextModelSnapshot.cs

```diff
@@ -117,6 +118,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasKey("Id");
 
+                    b.HasIndex("Type", "TenantId", "TenantType");
```

Comment:
Ho can you change Snapshot without changing DbContext schema and without creation migration?

Reply:
sorry, my fail. have pushed